From 0a29296ac0c1b60dbaa94183427102a2fb27f088 Mon Sep 17 00:00:00 2001
From: Kyle Horimoto <khorimoto@chromium.org>
Date: Tue, 16 Jun 2020 01:09:17 +0000
Subject: [cros search service] Add missing #include <cmath>

Fixes build for Clang 10 and libstdc++. Suggested by mike@fireburn.co.uk
on https://chromium-review.googlesource.com/c/chromium/src/+/2227721.

Change-Id: I110768027e4887be9566a965d62989f7e21ccdf6
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2242749
Auto-Submit: Kyle Horimoto <khorimoto@chromium.org>
Commit-Queue: Jia Meng <jiameng@chromium.org>
Reviewed-by: Jia Meng <jiameng@chromium.org>
Cr-Commit-Position: refs/heads/master@{#778591}
---
 chrome/common/string_matching/sequence_matcher.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/common/string_matching/sequence_matcher.cc b/chrome/common/string_matching/sequence_matcher.cc
index 6da316938d3f..c53bfc7acf0a 100644
--- a/chrome/common/string_matching/sequence_matcher.cc
+++ b/chrome/common/string_matching/sequence_matcher.cc
@@ -5,6 +5,7 @@
 #include "chrome/common/string_matching/sequence_matcher.h"
 
 #include <algorithm>
+#include <cmath>
 #include <queue>
 
 #include "base/check_op.h"
-- 
2.20.1

